src/Utils.cpp

```diff
@@ -772,6 +774,18 @@ bool CTable::AddColumn(const CString& sName) {
     return true;
 }
 
+bool CTable::SetStyle(const EStyle eNewStyle) {
+    switch (eNewStyle) {
```

Comment:
maybe "if"? switch here looks useless

Reply:
I think a switch is neater. It makes the intention of what SetStyle() does clearer: First, it checks if the requested style has any requirements (Gridstyle has none, ListStyle has 1). if all are fulfilled, it sets the style and returns true. 

If another style would get added to Estyle, it would be obvious that it should be added to the switch, but if there's only an `if (e==liststyle)`, it will look like liststyle is a special case.